src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content">
+    <div class="calendar page__calendar calendar--month-length--28 calendar--first-day--wed">
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
+      <div class="day calendar__day"></div>
```

Comment:
don't create unnecessary blocks, remove your `day` one

Reply:
![image](https://user-images.githubusercontent.com/118836537/216448172-39227eb3-94f1-415e-ac07-1056596e110f.png)
Not agree. If I remove them and write everything in one class. The code will be bulky and it will be uncomfortable for me to read and fix it. That's why I decided to break it down into smaller logical parts.